pages/api/loadNew.ts

```diff
@@ -0,0 +1,102 @@
+import * as fs from "fs";
+import path from "path";
+import { NextApiRequest, NextApiResponse} from 'next'
+
+import { getJsonStructure } from "evergreen-org-crawler/src/index"
+import config from "../../config.json"
+
+// Cache files are stored inside ./next folder
+const CachePath = path.resolve("./dynamicCache.json")
+const EmptyCachePath = path.resolve("./defaultDynamicCache.json")
+
+//TODO: Move to config file
+const timeUntilRefresh = 5 * 60 * 1000 // 5 minutes in milliseconds
+
+let waitingPromise: Promise<void> | null = null
+let promiseResolve: any = null
+
+function getProperty<T, K extends keyof T>(o: T, propertyName: K): T[K] {
+    return o[propertyName];
+}
+
+async function createData(request: "npm" | "PyPI" | "RubyGems" | null = null){
+	const requestToAPI = {
+		npm: "NPM",
+		PyPI: "PYPI",
+		RubyGems: "RUBYGEMS"
+	}
+
+	let api = null
+	if(request != null){
+		api = [getProperty(requestToAPI, request)]
+	}
+
+	const accessToken = process.env.NEXT_PUBLIC_GITHUB_TOKEN!
+	return getJsonStructure(accessToken, config, api)
+}
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+	let cachedData = null
+
+	const current = Date.now().valueOf()
```

Comment:
`valueOf()` is unnecessary, look at this [playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBGIHcYF4YBECGUCmA6eCAFAJQBQpokIANntSAOaEFmUQ12POK4BuG1AK7YA8gDMSxIA)